benefits/static/css/styles.css

```diff
@@ -178,6 +192,11 @@ footer {
   margin-bottom: 1rem;
   padding: 1rem;
   width: 100%;
+  text-transform: uppercase;
```

Comment:
This feels like the wrong place for the `text-transform` rule.

We want to transform `.btn-primary`, not necessarily every `.btn-lg` -- evidenced by the "undo" rule below for the `.btn-outline-primary`.